nautobot/core/templates/base.html

```diff
@@ -81,11 +108,12 @@ <h4><i class="mdi mdi-alert"></i> Maintenance Mode</h4>
                 </div>
                 <div class="col-xs-4 text-right noprint">
                     <p class="text-muted">
-                        <i class="mdi mdi-book-open-page-variant text-primary"></i> <a href="https://nautobot.readthedocs.io/">Docs</a> &middot;
+                        <i class="mdi mdi-book-open-page-variant text-primary"></i> <a href="{{ settings.BRANDING_URLS.docs }}">Docs</a> &middot;
                         <i class="mdi mdi-cloud-braces text-primary"></i> <a href="{% url 'api_docs' %}">API</a> &middot;
                         <i class="mdi mdi-graphql text-primary"></i> <a href="{% url 'graphql' %}">GraphQL</a> &middot;
-                        <i class="mdi mdi-xml text-primary"></i> <a href="https://github.com/nautobot/nautobot">Code</a> &middot;
-                        <i class="mdi mdi-lifebuoy text-primary"></i> <a href="https://github.com/nautobot/nautobot/wiki">Help</a>
+                        <i class="mdi mdi-xml text-primary"></i> <a href="{{ settings.BRANDING_URLS.code }}">Code</a> &middot;
+                        <i class="mdi mdi-lifebuoy text-primary"></i> <a href="{{ settings.BRANDING_URLS.help }}">Help</a> &middot;
+                        <i class="powered-by-nautobot text-primary"></i> <a href="{{ settings.BRANDING_POWERED_BY_URL }}">Powered</a>
```

Comment:
Oddly I had to shift-reload the page for this icon to appear correctly at first - initially it was missing.

Reply:
Sounds like normal browser caching or do think there is more there?

Reply:
Yeah, probably just caching or the like. Appears to render fine in Safari, Firefox, and Chrome now.